sdk/bootstrap/main.go

```diff
@@ -214,15 +212,20 @@ func openTemplate(inPath string, source string, templateMaxBytes int64) []byte {
 }
 
 func renderTemplate(origContent string, outPath string, envMap map[string]string, source string) {
-	dirpath, _ := path.Split(outPath)
-	template, err := mustache.ParseStringPartialsRawInDir(origContent, dirpath, nil, false)
-	if err != nil {
-		log.Fatalf("Failed to parse template content from %s at '%s': %s", source, outPath, err)
+	// Env preprocessing: map "false", "False", etc to a false bool, so that it's treated as 'falsy'.
+	varMap := make(map[string]interface{})
+	for k, v := range envMap {
+		if strings.ToLower(v) == "false" {
```

Comment:
trim whitespace?  I'm of two minds.  Maybe if you put whitespace around your booleans you deserve what you get?

Reply:
Yeah I originally had `TrimSpace()` here and then felt it was sloppy. Who knows, maybe someone will want `false ` to behave non-falsy.